nucleus/pom.xml

```diff
@@ -138,8 +138,8 @@
         <javax.el.version>3.0.1-b08</javax.el.version>
         <javax.el-api.version>3.0.1-b04</javax.el-api.version>
         <saaj-api.version>1.4.0</saaj-api.version>
-        <hk2.version>2.5.0-b44</hk2.version>
-        <hk2.plugin.version>2.5.0-b44</hk2.plugin.version>
+        <hk2.version>2.5.0-b57</hk2.version>
+        <hk2.plugin.version>2.5.0-b54</hk2.plugin.version>
```

Comment:
Please clarify why we are using 2 versions of HK2

Reply:
If there are any updates in HK2 code and not in plugins, they only release hk2 and plugins will retain the same old hk2 version. So, we have different versions for hk2 and hk2 plugins.